app/controllers/orders/new.js

```diff
@@ -6,6 +6,12 @@ export default Controller.extend({
       try {
         this.set('isLoading', true);
         let order = data;
+        let current_user = this.get('authManager.currentUser');
```

Comment:
current_user and order.user are the same things, just use a check for dirtyAttributes for the user model.

Reply:
@CosmicCoder96 on it

Reply:
Completed the required changes